bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,222 @@
+# MagentaTV Binding (aka TelekomTV aka EntertainTV)
+
+---
```

Comment:
Please add a short description of the binding here.

Reply:
done